samples/otlptracegrpc/app.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import express, {Express, Request, Response} from 'express';
+import {rollTheDice} from './dice';
+
+import {NodeSDK} from '@opentelemetry/sdk-node';
+import {OTLPTraceExporter} from '@opentelemetry/exporter-trace-otlp-grpc';
+import {AuthClient, GoogleAuth} from 'google-auth-library';
+import {credentials} from '@grpc/grpc-js';
+
+const PORT: number = parseInt(process.env.PORT || '8080');
+const app: Express = express();
```

Comment:
Prefer type inference over adding an annotation for most simple assignments https://google.github.io/styleguide/tsguide.html#type-inference

I think there are a few similar cases below